libcorn/src/lua.rs

```diff
@@ -0,0 +1,24 @@
+use crate::Value;
+use mlua::prelude::*;
+
+impl<'lua> ToLua<'lua> for Value<'lua> {
+    fn to_lua(self, lua: &'lua Lua) -> LuaResult<LuaValue<'lua>> {
+        Ok(lua.to_value(&self)?)
```

Comment:
There is a clippy warning here - since you're using the `? operator which returns a result anyway, you can just return the `lua.to_value` result directly.

```suggestion
        lua.to_value(&self)
```